README.md

```diff
@@ -273,3 +273,24 @@ following ways, in precedence order:
   in the case where you specify a path to a Python script via the `APP_FILE` environment
   variable, defaulting to a file named `app.py` if it exists. The script is
   passed to a regular Python interpreter to launch your application.
+
+Hot deploy
+---------------------
+
+If you are using Django, hot deploy will work out of the box.
```

Comment:
Yeah I just like the fact that first you are declaring that it's works OOTB and then then you actually tell to the user how to enable it for Gunicorn.